src/Propel/Generator/Schema/Dumper/XmlDumper.php

```diff
@@ -503,8 +503,10 @@ private function appendIndexNode(Index $index, \DOMNode $parentNode)
     /**
      * Appends the generated <unique> XML node to its parent node.
      *
-     * @param Unique   $unique     The Unique model instance
+     * @param Unique   $index
```

Comment:
It is a common practice for all library and framework code in PHP to have FQCN here, that will get rid of use statements that are only included to make doc blocks (semantics).